tasks/main.yml

```diff
@@ -37,25 +25,16 @@
 - include: postgres.yml
   when: install_postgres
 
-- name: Deploy Cabot from Git
-  git:
-    repo: https://github.com/arachnys/cabot.git
-    dest: '{{ cabot_config.app_path }}'
-    version: '{{ cabot.version }}'
-  become: yes
-  become_user: '{{ deploy_user }}'
-  tags:
-    - git
+- name: Install virtualenv
+  pip:
+    name: virtualenv
+  become: True
 
-- name: Install PIP modules
+- name: Install Cabot from pip
   pip:
-    requirements: '{{ cabot_config.app_path }}/{{ item }}'
+    name: "cabot=={{ cabot.version }}"
     virtualenv: '{{cabot_config.venv_path}}'
     virtualenv_site_packages: no
-  with_items:
-    - requirements.txt
-    - requirements-plugins.txt
```

Comment:
They'll be installed by pip (they are all part of the package requirements)

We need a system for custom plugins though